lib/iStats/cpu.rb

```diff
@@ -31,6 +31,9 @@ def cpu_temperature
         t = get_cpu_temp
         thresholds = [50, 68, 80, 90]
         value, scale = Printer.parse_temperature(t)
+        if scale == "#{Symbols.degree}F"
```

Comment:
This works but I'm wondering if we should use a getter on the printer's `temperature_scale` property instead of relying on the pretty printed symbol. Would just need a getter alongside the setter [here](https://github.com/Chris911/iStats/blob/2cf74aac6f44259da9d372c1f69ef5f5600de694/lib/iStats/printer.rb#L27-L29).

What do you think?

Reply:
You're completely right. It was my first thought but i couldn't wrap my head around the code.

Now it's very clear, thanks.